integration-cli/docker_cli_exec_test.go

```diff
@@ -43,37 +46,31 @@ func (s *DockerSuite) TestExecInteractiveStdinClose(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-
 	contId := strings.TrimSpace(out)
 
-	returnchan := make(chan struct{})
+	cmd := exec.Command(dockerBinary, "exec", "-i", contId, "echo", "-n", "hello")
+	p, err := pty.Start(cmd)
```

Comment:
We  extracted tests that has kr/pty into _unix.go files b/c pty does a lot of unix stuff to control the terminal and none of those stuff exists on windows (pkg doesnt even compile on windows). I don't think we can easily run kr/pty on windows @cpuguy83 but folks working on engine may come up with a solution there.